mungegithub/submit-queue/deployment/kops/configmap.yaml

```diff
@@ -0,0 +1,27 @@
+# basic config options.
+http-cache-dir: /cache/httpcache
+organization: kubernetes
+project: kops
+pr-mungers: lgtm-after-commit,submit-queue,needs-rebase
+state: open
+token-file: /etc/secret-volume/token
+period: 2m
```

Comment:
The loop period can be shorter than 2 minutes without adverse affects. Maybe start with 30 seconds?